### PR TITLE
chore: call aws secretsmanager

### PR DESCRIPTION
**Pull request by katebygrace**
JIRA:CLOUDSEC-12

### CODE REVIEW

File: dataeng/resources/model-transfers.sh [hunk 1]
```diff
@@ -28,7 +28,11 @@ fi
 
 ARGS="{mart: ${MART_NAME} }"
 
-dbt deps --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
+DBT_TARGET="prod"
```
Comment (HassanJaveed84) [inline dataeng/resources/model-transfers.sh:31]: Do we need to hardcode this to prod? 
DBT_TARGET is a jenkins job level parameter.
https://github.com/edx/jenkins-job-dsl/blob/a2d7d190f1576b6f8a6e76953957cc0f8fc8909a/dataeng/jobs/analytics/ModelTransfers.groovy#L22

Comment (katebygrace) [reply dataeng/resources/model-transfers.sh:31]: nah, though its prod everywhere in analytics secure but i can leave it

File: dataeng/resources/model-transfers.sh [hunk 2]
```diff
@@ -28,7 +28,11 @@ fi
...
+
+source secrets-manager.sh analytics-secure/warehouse-transforms/profiles DBT_PASSWORD
+
+dbt deps --profiles-dir . --profile $DBT_PROFILE --target $DBT_TARGET
```
[the remark this answers is not in the document]
Comment (katebygrace) [reply dataeng/resources/model-transfers.sh:33]: oh interesting - should i add the files below there?

File: dataeng/resources/model-transfers.sh [hunk 3]
```diff
@@ -28,7 +28,9 @@ fi
 
 ARGS="{mart: ${MART_NAME} }"
 
-dbt deps --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
+source secrets-manager.sh analytics-secure/warehouse-transforms/profiles DBT_PASSWORD
+
+dbt deps --profiles-dir $WORKSPACE/dataeng/profiles --profile $DBT_PROFILE --target $DBT_TARGET
```
Comment (HassanJaveed84) [inline dataeng/resources/model-transfers.sh:33]: So the workspace is a jenkins provided environment variable that points to the jenkins job workspace. 
jenkins-job-dsl repo. does not get cloned unless we specify it in the DSL.

**Pull request closed without merging**